### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by KangZhiDong**
Fix typo , ["An MBean is a managed Java object, similar to a JavaBeans component, that follows the design patterns set forth in the JMX specification. An MBean can represent a device, an application, or any resource that needs to be managed. MBeans expose a management interface that consists of the following:"](https://docs.oracle.com/javase/tutorial/jmx/mbeans/index.html )

### CODE REVIEW

**Comment by KangZhiDong** [comment]
@markt-asf  fix  some typos at the weekend  (●´∀｀●)ﾉ

**Comment by KangZhiDong** [comment]
@markt-asf  please review

**Comment by xiantang** [comment]
@KangZhiDong too much commits

**Comment by markt-asf** [comment]
Wow. There are a lot of changes here. Thank you.

It is too much to review in one go. I took a quick look at the first few changes files and things look good. My current plan is to review these in detail commit by commit and cherry-pick each commit once it has been reviewed.

I appreciate that some of these commits may depend on previous commits. That said, more PRs with fewer commits would be easier to work with where that is possible.

**Comment by markt-asf** [comment]
I'm about 2/3 of the way through the commits. With the benefit of experience, 1 PR vs multiple PRs doesn't make that much of a difference. What would have helped was re-basing the PR against master. I've done this locally and there were only a couple of conflicts and it makes it much easier to review.

I spotted a few unused i18n strings that weren't removed from the translations. That isn't a big deal as the process we use to import/export to/from POEditor will clean those up.

I have spotted a couple of minor issues. I'll fix those before merging the commits. I'll also comment on the commit in this PR where it makes sense to do so.

File: webapps/docs/windows-auth-howto.xml [hunk 1]
```diff
@@ -101,7 +101,7 @@ policy had to be relaxed. This is not recommended for production environments.
   &lt;service class&gt;/&lt;host&gt;:&lt;port&gt;/&lt;service name&gt;</code>.
   The SPN used in this how-to is <code>HTTP/win-tc01.dev.local</code>. To
   map the user to the SPN, run the following:
-  <source>setspn -A HTTP/win-tc01.dev.local tc01</source>
+  <source>setspn -An HTTP/win-tc01.dev.local tc01</source>
   </li>
```
Comment (markt-asf) [inline webapps/docs/windows-auth-howto.xml]: This change is incorrect. This is a command line option hence it needs to be `.. -A HTTP...`

Comment (xiantang) [reply webapps/docs/windows-auth-howto.xml]: > I'm about 2/3 of the way through the commits. With the benefit of experience, 1 PR vs multiple PRs doesn't make that much of a difference. What would have helped was re-basing the PR against master. I've done this locally and there were only a couple of conflicts and it makes it much easier to review.
> 
> I spotted a few unused i18n strings that weren't removed from the translations. That isn't a big deal as the process we use to import/export to/from POEditor will clean those up.
> 
> I have spotted a couple of minor issues. I'll fix those before merging the commits. I'll also comment on the commit in this PR where it makes sense to do so.

lol ! It is so hard to review

**Comment by KangZhiDong** [comment]
It seems to have been merged

**Pull request closed without merging**